README.md

```diff
@@ -58,6 +58,12 @@ As the API has just launched, not many implementations exist yet. If you are int
 - [StatsD Client](https://github.com/apple/swift-statsd-client), support for StatsD
 - Your library? [Get in touch!](https://forums.swift.org/c/server)
 
+### Swift Metrics Extras
+
+You may also be interestedin some "extras" modules which are collected in the [swift metrics extras](https://github.com/apple/swift-metrics-extras) repository.
+
+Currently the repository includes: `SystemMetrics` allowing reporting of a node's CPU and memory metrics etc.
```

Comment:
```suggestion
Currently the repository includes `SystemMetrics` which allows reporting of a node's CPU and memory metrics etc.
```

Reply:
not sure its worth detailing what extra includes as it won't scale to update both places. maybe just mention the kind of things that extra may include.

Reply:
Yeah that's a concern -- I'll just link to it then